src/Kopernicus.Components/KopernicusStar.cs

```diff
@@ -155,10 +155,10 @@ public static Double Flux(ModularFlightIntegrator fi, KopernicusStar star)
                 // Get sunVector
                 RaycastHit raycastHit;
                 Boolean directSunlight = false;
-                Vector3d scaledSpace = ScaledSpace.LocalToScaledSpace(fi.IntegratorTransform.position);
+                Vector3d scaledSpace = ScaledSpace.LocalToScaledSpace(fi.Vessel.CoMD);
```

Comment:
Is CoMD actually the same as the vessel position in world space? I am not entirely sure, and can't check it at the moment.